apps/docs/pages/guides/getting-started/tutorials/with-react.mdx

```diff
@@ -168,7 +168,7 @@ export default function Account({ session }) {
       id: user.id,
       username,
       website,
-      avatar_url,
+      avatarUrl ?? avatar_url,
```

Comment:
Do we need `avatar_url` anymore?

Reply:
@gregnr This PR made a change that is reflected later in the tutorial where Avatar is used. Here:
<img width="609" alt="Screenshot 2023-06-30 at 23 40 25" src="https://github.com/supabase/supabase/assets/14860945/c85639c2-ecf6-47f1-ad82-b6519ee42e35">

However, the site where updateProfile is declared, it only takes in a single parameter. Here:
<img width="626" alt="Screenshot 2023-06-30 at 23 40 52" src="https://github.com/supabase/supabase/assets/14860945/1414d1e4-63bd-40b4-8fed-300950a87f0a">

This is why I made this change, so that it doesn't cause confusion. So that it is clear that updateProfile takes in two parameters and uses the one from state only when it is available.

Reply:
Right, I agree with this - but if `updateProfile()` always expects an `avatarUrl` now, the fallback `avatar_url` should be unnecessary right?